tools/asc-local-migration/asc-2-asc-local.ts

```diff
@@ -15,8 +15,8 @@ const NODE_MODULES_IGNORE_LINE = 'node_modules/';
 const CPLACE_ASC_LOCAL_PKG_NAME = '@cplace/asc-local';
 const CPLACE_ASC_LOCAL_PKG_VERSION = '^2.0.7';
 const PACKAGES_TO_REMOVE = ['typescript'];
-const NODE_VERSION = '18.11.0';
-const NPM_VERSION = '8.19.2';
+const NODE_VERSION = '18.18.2';
```

Comment:
Can't we use `SupportedNodeVersions.SUPPORTED_NODE_VERSION` for that? That way we'd have one place less to change next time. 👀